user.go

```diff
@@ -52,8 +52,8 @@ type User struct {
 	// The user's username.
 	Username string `json:"username"`
 
-	// The hash of the user's avatar. Use Session.UserAvatar
-	// to retrieve the avatar itself.
+	// The hash of the user's avatar. Use Session.UserAvatarDecode
+	// to retrieve the avatar itself (Session.UserAvatar is deprecated).
```

Comment:
I think it's better just to replace the function.
Also we should probably mention the `AvatarURL` here.
```suggestion
	// The hash of the user's avatar.
	// Use AvatarURL to retrieve the avatar URL or Session.UserAvatarDecode to retrieve the avatar itself.
```